From mitov@issp.bas.bg  Tue Jun 15 12:04:02 2010
From: Marin Mitov <mitov@issp.bas.bg>
Date: Mon, 31 May 2010 13:03:04 +0300
Subject: Driver core: internal struct dma_coherent_mem, change type of a member.
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Cc: linux-kernel@vger.kernel.org
Message-ID: <201005311303.04788.mitov@issp.bas.bg>
Content-Disposition: inline


struct dma_coherent_mem in drivers/base/dma-coherent.c
has member 'device_base' that is of type u32,
but is assigned value of type dma_addr_t, which may be
64 bits for x86_64. Change the type to dma_addr_t.

Signed-off-by: Marin Mitov <mitov@issp.bas.bg>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/dma-coherent.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/base/dma-coherent.c
+++ b/drivers/base/dma-coherent.c
@@ -8,7 +8,7 @@
 
 struct dma_coherent_mem {
 	void		*virt_base;
-	u32		device_base;
+	dma_addr_t	device_base;
 	int		size;
 	int		flags;
 	unsigned long	*bitmap;
